doc/source/conf.py

```diff
@@ -103,7 +108,21 @@
 # further.  For a list of options available for each theme, see the
 # documentation.
 #
-# html_theme_options = {}
+html_theme_options = {
+    'logo': 'kymatio.jpg',
```

Comment:
Thought we were going to do SVG here?

Reply:
@janden the JPG one is 16kb so.. way lighter!

Reply:
:rofl: